Tests/AsyncHTTPClientTests/HTTPClientPerformanceBenchmarkTests.swift

```diff
@@ -0,0 +1,117 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the AsyncHTTPClient open source project
+//
+// Copyright (c) 2018-2019 Apple Inc. and the AsyncHTTPClient project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of AsyncHTTPClient project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+//
+// HTTPClientPerformanceBenchmarkTests.swift.swift
+import AsyncHTTPClient
+import Foundation
+import XCTest
+
+private class AsyncHTTPClientTestOperation: AsynchronousOperation {
+    private let httpClient: HTTPClient
+    
+    init(httpClient: HTTPClient) {
+        self.httpClient = httpClient
+        super.init()
+    }
+    
+    override func execute() {
+        httpClient.get(url: "https://swift.org").whenComplete { _ in
```

Comment:
we shouldn't reach out to the internet during benchmarks, you'd be benchmarking mostly the network and not the library then.